subprojects/scala/src/integTest/groovy/org/gradle/scala/compile/IncrementalScalaCompileIntegrationTest.groovy

```diff
@@ -36,12 +35,8 @@ class IncrementalScalaCompileIntegrationTest extends AbstractIntegrationSpec {
             compileScala.options.debug = false
 '''
         then:
-        // This gets reset each time you run() is run.
-        executer.expectDeprecationWarning();
```

Comment:
I guess the deprecation warnings where emitted by the Ant Scala compiler which we removed for 4.0. Surprisingly (to me), `expectDeprectationWarning()` doesn't fail when no deprecation warning is emitted. Therefore, this code was still in place but unnecessary.